packages/js/e2e-core-tests/package.json

```diff
@@ -8,12 +8,26 @@
     "url": "https://github.com/woocommerce/woocommerce.git"
   },
   "license": "GPL-3.0+",
-  "main": "index.js",
+  "main": "build/index.js",
+  "module": "build-module/index.js",
   "dependencies": {
-    "@jest/globals": "^26.4.2",
+    "@jest/globals": "^26.6.2",
```

Comment:
Unless we need to bump packages to complete a PR, we do package bumps on package bump PRs. This will become more important when WC Admin is added to the repo and we have ~20 `npm` packages where we are trying to sync dependency versions. Please revert these two dependency bumps.